package.json

```diff
@@ -0,0 +1,14 @@
+{
+    "name": "gage-conditions",
+    "version": "",
+    "description": "",
+    "scripts": {
+        "start": "webpack --mode development"
+    },
+    "dependencies": {
+        "webpack": "^4.12.0"
+    },
+    "devDependencies": {
+        "webpack-cli": "^3.0.8"
```

Comment:
Its odd we have `webpack` as a dependency and `webpack-cli` as only a dev dependency.  We need to have both to have webpack work, so I feel a stronger solution would be to have the 2 packages in both the production dependencies and the dev dependencies.  Which is super easy to do as you are installing them with node.  Just simply run `npm i webpack webpack-cli --save --save-dev` or have them both in the production line only, I just don't like them split up as they currently are.

Reply:
Not that it matters for this specific test project, just in the future let's make sure they are together.

Reply:
Tried to troubleshoot a little with marty. No luck, so we created #8 for now.